modules/statics/src/coins/erc20Coins.ts

```diff
@@ -1,5 +1,58 @@
-import { UnderlyingAsset } from '../base';
-import { erc20 } from '../account';
+import { CoinFeature, UnderlyingAsset } from '../base';
+import { AccountCoin, erc20 } from '../account';
+import { MATIC_FEATURES, WETH_FEATURES, ETH_FEATURES } from './coins';
```

Comment:
Isn't it supposed to be 
```suggestion
import { MATIC_FEATURES, WETH_FEATURES, ETH_FEATURES } from '../coins';
```